### PR TITLE
feat: add phone international field

### PR DESCRIPTION
**Pull request by coppee**
|Q              |A  |
|---------------|---|
|Bug fix?       |N|
|New feature?   |Y|	
|BC breaks?     |N|	
|Deprecations?  |N|	
|Fixed tickets  |N|

### CODE REVIEW

File: src/Components/Field/InternationalPhone.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
```
Comment (Davidmattei) [inline src/Components/Field/InternationalPhone.php:2]:
```suggestion
declare(strict_types=1);
```

File: src/Components/Field/InternationalPhone.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    public function getFieldClass(): string
+    {
+        return TelType::class;
+    }
+
+    public function getOptions(): array
+    {
+        $options = parent::getOptions();
+
+        if (!empty($this->config->getChoiceList())) {
```
Comment (Davidmattei) [inline src/Components/Field/InternationalPhone.php:24]: No empty \count

**Review by Davidmattei — CHANGES_REQUESTED** [review]

File: src/Components/Field/InternationalPhone.php (hunk 1 above)
Comment (coppee) [reply src/Components/Field/InternationalPhone.php:2]: Done

File: src/Components/Field/InternationalPhone.php (hunk 2 above)
Comment (coppee) [reply src/Components/Field/InternationalPhone.php]: Done

**Review by theus77 — APPROVED** [review]

**Review by Davidmattei — APPROVED** [review]

**Pull request merged**